src/main/java/com/artipie/api/UsersRest.java

```diff
@@ -33,15 +35,33 @@ public void init(final RouterBuilder rbr) {
         rbr.operation("listAllUsers")
             .handler(this::listAllUsers)
             .failureHandler(this.errorHandler(HttpStatus.INTERNAL_SERVER_ERROR_500));
+        rbr.operation("getUser")
+            .handler(this::getUser)
+            .failureHandler(this.errorHandler(HttpStatus.INTERNAL_SERVER_ERROR_500));
+    }
+
+    /**
+     * Get single user info.
+     * @param context Request context
+     */
+    private void getUser(final RoutingContext context) {
+        final Optional<JsonValue> usr = this.users.list().stream().filter(
```

Comment:
can we add method CrudUsers.user(final String name) and use it?
Because we convert all Yaml-users to JsonArray and afteward search in JsonArray, I seem it is memory consumption.